### PR TITLE
Add OMFIT dependencies packages

### PR DESCRIPTION
**Pull request by smithsp**
@namurphy What is the best way to test this?

### CODE REVIEW

**Comment by namurphy** [comment]
> @namurphy What is the best way to test this?

If we go to [mybinder.org](https://mybinder.org/) then we can copy the link to the repository, specify the branch, and then launch an environment.  I copied this button over so hopefully it'll work!

[Binder](https://mybinder.org/v2/gh/PlasmaPy/hack-week-environment-2021/omfit_deps)

**Comment by namurphy** [comment]
Got this problem...

> `Problem: nothing provides gpr1dfusion >= 1.0`

File: environment.yml [hunk 1]
```diff
@@ -20,6 +21,8 @@ dependencies:
   - tofu
   - omas
   - xbout
+  - omfit-opt-deps
+  - omfit_classes
   - pip:
     - turbopy
```
Comment (namurphy) [inline environment.yml:27]:
```suggestion
    - gpr1dfusion
    - turbopy
```
My best guess is that having it installed by `pip` would suffice.

Comment (smithsp) [reply environment.yml:28]: Good try, but this will still fail the conda solve.  I have added the correct channel that is needed (`aaronkho`).

**Comment by namurphy** [comment]
I just tried running `conda env create -f environment.yml` on this `environment.yml` file...
```yaml
name: hackweek533
channels:
  - conda-forge
  - defaults
  - omfit
  - aaronkho
dependencies:
  - python=3.7.9
  - plasmapy=0.6.0
  - omfit-opt-deps=3.2021.20
  - omfit_classes
```
and got the following information about package conflicts.
```
Collecting package metadata (repodata.json): | WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
done
Solving environment: \ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                        
Solving environment: - 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                        

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

Package python conflicts for:
python=3.7.9
plasmapy=0.6.0 -> astropy[version='>=3.1'] -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.5,<3.6.0a0|>=3.6|>=3.5|>=3|3.4.*|>=2.7']
plasmapy=0.6.0 -> python[version='>=3.7']
omfit_classes -> python[version='>=3.7']

Package python-dateutil conflicts for:
omfit-opt-deps=3.2021.20 -> botocore[version='>=1.8'] -> python-dateutil[version='>=2.1,<2.7.0|>=2.1,<2.8.1|>=2.1,<3.0.0']
omfit-opt-deps=3.2021.20 -> python-dateutil[version='>=1.5']

Package numpy conflicts for:
omfit-opt-deps=3.2021.20 -> aurorafusion[version='>=0.3'] -> numpy[version='*|>=1.10|>=1.10.4|>=1.11.0|>=1.13.0|>=1.15.1|>=1.16|>=1.17.5,<2.0a0|>=1.19.5,<2.0a0|>=1.18.5,<2.0a0|>=1.20.1,<2.0a0|>=1.16.5,<2.0a0|>=1.19.2,<2.0a0|>=1.14.6,<2.0a0|>=1.9.3,<2.0a0|>=1.11|>=1.13.3|>=1.9|>=1.9.1|>=1.8|1.13.*|1.12.*|1.11.*|>=1.11.3,<2.0a0|>=1.19.4,<2.0a0|>=1.16.6,<2.0a0|>=1.14.6,<1.18.0|1.15.*|>=1.15.4,<2.0a0|>=1.12,<1.20|>=1.11,<2.0a0|>=1.15.2,<2.0a0|>=1.13.3,<2.0a0|1.10.*|>=1.9.3|>=1.7|1.9.*|1.8.*|>=1.6.1']
plasmapy=0.6.0 -> numpy[version='>=1.20.1,<2.0a0']
plasmapy=0.6.0 -> astropy[version='>=3.1'] -> numpy[version='1.11.*|1.12.*|1.13.*|>=1.11|>=1.11.3,<2.0a0|>=1.13.3,<2.0a0|>=1.13|>=1.14|>=1.16.5,<2.0a0|>=1.16|>=1.17.5,<2.0a0|>=1.17|>=1.19.5,<2.0a0|>=1.19.4,<2.0a0|>=1.19|>=1.16.6,<2.0a0|>=1.17.0,<2.0a0|>=1.17.4,<2.0a0|>=1.14.6,<2.0a0|>=1.19.2,<2.0a0|>=1.9.3,<2.0a0|>=1.8|>=1.18|>=1.18.5,<2.0a0|>=1.15.4,<2.0a0|>=1.18.4,<2.0a0|>=1.18.1,<2.0a0|>=1.9|>=1.15.1,<2.0a0|>=1.15']

Package tzdata conflicts for:
plasmapy=0.6.0 -> python[version='>=3.7'] -> tzdata
omfit_classes -> python[version='>=3.7'] -> tzdata
omfit-opt-deps=3.2021.20 -> postgresql[version='>=9.6'] -> tzdata

Package freetype conflicts for:
omfit-opt-deps=3.2021.20 -> ffmpeg[version='>=1.4'] -> freetype[version='2.8.1|2.8.1.*|>=2.10.4,<3.0a0|>=2.9.1,<3.0a0|>=2.8.1,<2.9.0a0|>=2.8.1,<2.8.2.0a0|>=2.8,<2.9.0a0|>=0.0.*']
plasmapy=0.6.0 -> matplotlib-base[version='>=2.0'] -> freetype[version='>=2.10.2,<3.0a0|>=2.10.4,<3.0a0|>=2.9.1,<3.0a0']
python=3.7.9 -> tk[version='>=8.6.10,<8.7.0a0'] -> freetype[version='>=2.10.1,<3.0a0']
```
Right now I'm wondering if having a single `environment.yml` file and binder environment for the whole hack week is too ambitious given conflicting requirements for different software packages.  Maybe we ought to split this up into a few different `environment.yml` files...and then make sure to talk about how to use Anaconda Navigator on the first day and switch between environments.  Hm...

**Comment by smithsp** [comment]
@namurphy Thanks for looking into this.  I had been opening the binder link you gave each time, and had failed to follow up with the latest conflicts.  Probably having separate environments is the way to go.

**Comment by rocco8773** [comment]
Closing since this was a part of the 2021 Hack Week.

**Pull request closed without merging**